runtimes/common/src/benchmarks.rs

```diff
@@ -27,3 +34,67 @@ impl<A> From<(A, Vec<u8>)> for DummySignature {
 		DummySignature
 	}
 }
+
+pub mod treasury {
+	use super::*;
```

Comment:
Le't not use `use super::*`, they're less than ideal to deal with.

Reply:
[f7e5129](https://github.com/KILTprotocol/kilt-node/pull/655/commits/f7e51296002107e04d22f4317f75d4b3a6822194)